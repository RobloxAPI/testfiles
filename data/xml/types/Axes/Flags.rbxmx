<roblox version="4">
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">FFF</string>
			<Axes name="Axes">
				<axes>0</axes>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">TFF</string>
			<Axes name="Axes">
				<axes>1</axes>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">FTF</string>
			<Axes name="Axes">
				<axes>2</axes>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">TTF</string>
			<Axes name="Axes">
				<axes>3</axes>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">FFT</string>
			<Axes name="Axes">
				<axes>4</axes>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">TFT</string>
			<Axes name="Axes">
				<axes>5</axes>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">FTT</string>
			<Axes name="Axes">
				<axes>6</axes>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">TTT</string>
			<Axes name="Axes">
				<axes>7</axes>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">Multiple</string>
			<Axes name="Axes">
				<axes>2</axes>
				<axes>3</axes>
			</Axes>
		</Properties>
	</Item>
</roblox>
