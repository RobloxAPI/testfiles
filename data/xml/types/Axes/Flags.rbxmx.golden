{
	"Format": "rbxmx",
	"Output": "model",
	"Data": {
		"Metadata": [],
		"Instances": [
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 0,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "FFF"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 1,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": true,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "TFF"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 2,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": true,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "FTF"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 3,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": true,
							"Y": true,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "TTF"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 4,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": true
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "FFT"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 5,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": true,
							"Y": false,
							"Z": true
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "TFT"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 6,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": true,
							"Z": true
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "FTT"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 7,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": true,
							"Y": true,
							"Z": true
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "TTT"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 8,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": true,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "Multiple"
					}
				],
				"Children": []
			}
		]
	}
}